app/helpers/service_helpers.py

```diff
@@ -0,0 +1,4 @@
+def get_lot_name_from_acronym(blueprint, acronym):
+    """Return the full lot name for its acronym"""
+
+    return blueprint.config['LOTS'][acronym]
```

Comment:
This could throw a `KeyError` which isn't being handled at the call site. What's the intended behaviour if the acronym isn't found?